src/types/options.js

```diff
@@ -5,7 +5,8 @@ export default PropTypes.arrayOf(
   PropTypes.oneOfType([
     PropTypes.shape({
       value,
-      label: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
+      label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+      disabled: PropTypes.boolean
```

Comment:
Will this beef if undefined?

Reply:
I assumed so

Reply:
oh sorry, you mean if disabled is undefined, no, no beef, it has a .isRequired prop to make it beef if undefined